package.json

```diff
@@ -6,7 +6,6 @@
         "lint:staged": "lerna run lint:staged --stream",
         "lint": "lerna run lint --stream",
         "manual-version": "lerna version --preid=rc --no-push --no-changelog --no-private",
```

Comment:
would it be better to update to `version:rc` as the script will only version with preid `rc`?

Reply:
sure